src/pages/Payment/Payment.js

```diff
@@ -0,0 +1,197 @@
+              </Text>
+            </Div>
+            <Div column>
+              <Text margin="0 0 8px 0" color="gray" fontSize="11px">
+                5개월 할부 시 월 2000원
+              </Text>
+              <Title fontSize="16px" fontWeight="700">
+                1111원
+              </Title>
+            </Div>
+          </Section>
+          <Section>
+            <Title>결제 방식</Title>
+            <Div marginTop>
+              <Div>
+                <Button border bgColor="white" margin width="288px">
+                  weBank
+                </Button>
+                <Button border bgColor="white" margin width="288px">
+                  달란트
+                </Button>
+              </Div>
+              <Div>
+                <Button border bgColor="white" margin width="288px">
+                  기프티콘
+                </Button>
+                <Button border bgColor="white" margin width="288px">
+                  포인트
+                </Button>
+              </Div>
+            </Div>
+          </Section>
+          <Button fontWeight color="white" bgColor="#FF7129">
+            결제하기
+          </Button>
+        </SectionWrap>
+      </PaymentWrap>
+    </Body>
+  );
+};
+
+const Body = styled.div`
+  background-color: #f8f8f9;
+`;
+const PaymentWrap = styled.div`
+  width: 640px;
+  height: 1000px;
+  margin: 0 auto;
+  padding-top: 80px;
+  padding-bottom: 100px;
+  background-color: white;
+`;
+
+const SectionWrap = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 100%;
+  padding: 24px;
+`;
+const Header = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 84px;
+  padding: 24px;
+  color: white;
+  background-color: black;
+`;
+
+const Section = styled.section`
+  width: 100%;
+  height: ${props => props.height};
+  display: flex;
+  flex-direction: ${props => (props.row ? "row" : "column")};
+  justify-content: space-between;
+`;
+
+const Div = styled.div`
+  display: flex;
+  flex-direction: ${props => (props.column ? "column" : "row")};
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: ${props => (props.marginTop ? "12px" : "0")};
+`;
+
+const Img = styled.img`
+  width: 90px;
+  height: 72px;
+  border: 1px solid black;
+`;
+
+const Title = styled.h2`
+  font-size: ${props => (props.fontSize ? "26px" : "20px")};
+  font-weight: ${props => (props.fontWeight ? "700" : "900")};
+`;
+
+const Text = styled.span`
+  color: ${props => props.color};
+  font-size: ${props => props.fontSize};
+  font-weight: ${props => props.fontWeight};
+  margin: ${props => props.margin};
+`;
+
+const CoupontInput = styled.input`
+  width: 500px;
+  padding: 13px 16px;
+`;
+
+const Button = styled.button`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  outline: black;
+  border: ${props => (props.border ? "1px solid lightgray" : "none")};
```

Comment:
```suggestion
  border: ${(border}) => (border ? "1px solid lightgray" : "none")};
```
* 이렇게 반복되는 props는 구조분해할당 해 줄 수 있습니닷!

Reply:
오옷 반영했습니다!